packages/core/src/components/tag/_common.scss

```diff
@@ -70,7 +71,7 @@ $tag-round-adjustment: 2px !default;
   line-height: $tag-line-height-large;
   min-height: $tag-height-large;
   min-width: $tag-height-large;
-  padding: 0 $tag-padding-large;
+  padding: $tag-padding-large-top $tag-padding-large;
```

Comment:
can you just use `($tag-padding-large / 2)` here instead? I'd prefer that over adding a new Sass variable